base/inc/AdePT/BVHNavigator.h

```diff
@@ -225,6 +225,8 @@ class BVHNavigator {
 
     VPlacedVolumePtr_t pvol = state.Top();
 
+    if (!pvol) return;
+
```

Comment:
I still maintain that this isn't needed: If there's no current volume, there's nothing to relocate and the function shouldn't be called in the first place.

Reply:
Without this, the code crashes.

Reply:
It won't crash if you move the call (see other comment)

Reply:
That's doesn't matter, it *can* crash if the point is outside the geometry, and that should not crash, even if we move this call and in this particular example it would no longer crash.

Reply:
My point is that calling the function is wrong, same as calling `memcpy` with an invalid pointer. But if you argue that way, it should be in a separate PR and also be changed in the `LoopNavigator`.

Reply:
Fine, I removed this change.